src/styles/blocks/root.scss

```diff
@@ -0,0 +1,9 @@
+body {
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
Setting a fixed width on the `body` element is not a common practice and can lead to unexpected layout issues. It's better to control the width of content through container elements or through responsive design techniques.